practice/golang_learning_test.go

```diff
@@ -3,47 +3,63 @@ package practice
 import (
 	"context"
 	"github.com/stretchr/testify/assert"
+	"strings"
+	"sync"
 	"testing"
 	"time"
 )
 
 // golang 학습 테스트
 func TestGolang(t *testing.T) {
 	t.Run("string test", func(t *testing.T) {
-		//str := "Ann,Jenny,Tom,Zico"
-		//actual := "" // TODO str을 , 단위로 잘라주세요.
-		//expected := []string{"Ann","Jenny","Tom","Zico"}
-		//TODO assert 문을 활용해 actual과 expected를 비교해주세요.
+		str := "Ann,Jenny,Tom,Zico"
+		actual := strings.Split(str, ",")
+		expected := []string{"Ann", "Jenny", "Tom", "Zico"}
+		assert.Equal(t, expected, actual)
 	})
 
 	t.Run("goroutine에서 slice에 값 추가해보기", func(t *testing.T) {
+		const SIZE int = 100
```

Comment:
golang에서는 const를 대문자로 쓰지 않는 편입니다.
```suggestion
		const size int = 100
```